apps/src/storage/dataBrowser/column-header.module.scss

```diff
@@ -0,0 +1,24 @@
+$maxCellWidth: 350px;
+
+.columnName {
+    display: inline-block;
+    max-width: $maxCellWidth;
+    overflow: hidden;
+    white-space: nowrap;
+};
```

Comment:
a couple of nits: add an empty newline between class definitions (this should be solved for you once we add linting!) and remove the `;`

Reply:
Thanks! I'll make a pass on all 6 modules for this.